yarn-project/noir-protocol-circuits/src/crates/private-kernel-lib/src/private_kernel_init.nr

```diff
@@ -619,4 +622,23 @@ mod tests {
         // non-transient read requests are NOT forwarded
         assert_eq(array_length(public_inputs.end.read_requests), MAX_READ_REQUESTS_PER_CALL);
     }
+
+    #[test]
+    fn propogate_nullifier_key_validation_requests() {
```

Comment:
```suggestion
    fn propagate_nullifier_key_validation_requests() {
```
naming nit!

Reply:
Hahaha very impressive that you can always these!